test/metabase/analytics/snowplow_test.clj

```diff
@@ -36,13 +36,13 @@
         context (->> event .getContext first .getMap normalize-map)]
     (swap! *snowplow-collector* conj {:properties properties, :subject subject, :context context})))
 
-(defn- do-with-fake-snowplow-collector
+(defn do-with-fake-snowplow-collector
   "Impl for `with-fake-snowplow-collector` macro; prefer using that rather than calling this directly."
   [f]
-  (mt/with-temporary-setting-values [snowplow-available true]
-    (binding [*snowplow-collector* (atom [])]
```

Comment:
This binding was unnecessary and was preventing the macro from working in `metabase.api.setup-test` since the Snowplow logging was happening in a different thread from the test.

Reply:
🤯 that's smart, let me try it